qemu/tests/cfg/iperf_test.cfg

```diff
@@ -43,3 +43,12 @@
                 - host2guest:
                 - guest2host:
                     iperf_server = ${main_vm}
+        - with_short_packets:
+            iperf_version = iperf-2.1.9
+            host_iperf_file = ${iperf_version}.tar.gz
+            Windows:
+                guest_iperf_file = ${iperf_version}.exe
+            catch_data = '%s connected to %s'
```

Comment:
`[  1] local 10.72.136.74 port 5001 connected with 10.72.138.76 port 37503 (icwnd/mss/irtt=14/1448/95)` 
The actul log are different than what you will capture, so please fix it.

Reply:
@yanglei-rh, I have fixed it. Could you please help review it again?

Reply:
Is there any reason setup different command parameter for server and client? If not, the better is to keep command's consystency.

Reply:
I think they are the minimal parameters, the following is the reason.
```
-c 10.73.72.182: This specifies the server IP address the iperf client will connect to (10.73.72.182).
-B 10.73.74.248: This binds the iperf client to the local IP address 10.73.74.248.
-l 2: This sets the data packet length to 2 bytes.
-t 60: This runs the test for 60 seconds.
```

Reply:
No, you don't got the my point. Please review your command again.